collaboration_proposal.md

```diff
@@ -0,0 +1,152 @@
+        - Each collaborator defines the respective method he/she wants to primarily evaluate in the simulation study.
+    ** Step 2: Merged Simulation Study: **
+        - directly inherited from step 1.
+
+6. Defining Estimands / Population level values
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - Result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+    - Perspective Swapping, Core Disagreements, Listing
+    - Red-Teaming: what if scenarios (Kardos & Dexter, 2017)
+        - Generate and test alternative estimands with the goal of invalidating the arbiters decisions.
+
+7. Performance Measures
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - Result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+        - Red-Teaming: what if scenarios (Kardos & Dexter, 2017:
+            - Generate alternative performance measures for which the arbiters method fails or underperforms.
+            - Generate alternative scenarios (i.e. modulating each other step) for which the arbiters chosen performance measure fails or underperforms.
+
+8. Software Selection
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - Result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+        - Red-Teaiming: Possible but (probably) practially unapplicable and ineffective:
+
+9. Coding and Execution
+    ** Step 1: Individual Simulation Studies: **
+        - Conducted independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - One script as result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+        - Critically: A list of coding decisions that have implications for the results of the simulation study should be compiled and exposed to the adversarial process.
+
+10. Analyzing results
+    ** Step 1: Individual Simulation Studies: **
+        - Chosen independently by each collaborator.
+
+    ** Step 2: Merged Simulation Study: **
+        - Result of the adversarial process.
+
+    ** Adversarial Collaboration Techniques used: **
+        - Perspective Swapping, Core Disagreements, Listing
+        - Red-Teaming: what if scenarios (Kardos & Dexter, 2017):
+            - Generate alternative scenarios that invalidates the arbiters analysis decisions 
+            - Generate alternative analysis methods that show that the arbiters method fails / underperforms.
```

Comment:
I feel like this will be too late at this point. The results will be agreed upon in advance and ideally, we do not question this so late on in the process. I am afraid of constantly reoccuring arguments about the same thing. This also links to my previous thought with regards to Red-Teaming: I think in this section, both sides will rather focus on how to portray their own method in a more favorable light, rather than discrediting the other.